apps/admin_api/lib/admin_api/v1/plugs/idempotency.ex

```diff
@@ -0,0 +1,31 @@
+defmodule AdminAPI.V1.Plug.Idempotency do
```

Comment:
Can we merge with `EWalletAPI.V1.Plug.Idempotency` and move to `EWallet.Web.V1.IdempotencyPlug`?

Reply:
Tried it... But had some issues with it so I moved back to duplicated code.